_layouts/posts.html

```diff
@@ -32,7 +32,14 @@
                 {% if post.external %}
                     <a href="{{ post.external }}"><p>{{ post.title | escape }}</p></a>
                 {% else %}
-                    <a href="{{ post.url }}"><p>{{ post.title | escape }}</p></a>
+                    <a href="{{ post.url }}">
+                    <p>{{ post.title | escape }}
```

Comment:
![image](https://github.com/open-life-science/open-life-science.github.io/assets/1842467/fed215ba-f76e-4740-8489-13a917cfbcd5)

It looks strange. The subtitle should be small than the title. So probably 2 different `<p>` elements, with different classes